benchmark/arxiv_kaggle/data_generation.py

```diff
@@ -0,0 +1,142 @@
+import json
+import shutil
+from pathlib import Path
+from typing import Annotated, Any
+
+import pandas as pd
+import typer
+from benchmark.utils.time_resolution_binning import (
+    HELP_RESOLUTION,
+    bin_dataframe_wrt_time_resolution,
+    df_to_csv_with_timestamp,
+)
+from modyn.const.types import TimeResolution
+from sklearn.model_selection import train_test_split
+from torch.utils.data import Dataset
+from tqdm import tqdm
+
+
+class ArxivKaggleDownloader(Dataset):
```

Comment:
Not sure we need to inherit from dataset here. Dataset is a class to be put into a data loader for training. Reading the code, I don't think we need that, do we?

Reply:
No, that's simply an artifact of copying the baseline of another `data_generation.py` script.